From: Jiri Kosina <jkosina@suse.cz>
Subject: [PATCH] Revert "mkuboot.sh: Fail if mkimage is missing"
Patch-mainline: no

This reverts upstream commit bc91c9f3 ("mkuboot.sh: Fail if mkimage is
missing") as it causes the whole build to be aborted when mkimage command
is missing.
---
 scripts/mkuboot.sh |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/scripts/mkuboot.sh b/scripts/mkuboot.sh
index 446739c..2e3d3cd 100755
--- a/scripts/mkuboot.sh
+++ b/scripts/mkuboot.sh
@@ -11,7 +11,7 @@ if [ -z "${MKIMAGE}" ]; then
 	if [ -z "${MKIMAGE}" ]; then
 		# Doesn't exist
 		echo '"mkimage" command not found - U-Boot images will not be built' >&2
-		exit 1;
+		exit 0;
 	fi
 fi
 
-- 
1.7.5.4

